.pre-commit-config.yaml

```diff
@@ -37,8 +37,13 @@ repos:
     rev: v1.4.1
     hooks:
       - id: mypy
-        # To avoid dependencies redefinition.
+        name: run mypy
```

Comment:
it should be `local` hook, please check [my comment](https://github.com/AlexMomot-717/python-project-lvl2/pull/4#discussion_r1303572470) once again :)